kore/package.yaml

```diff
@@ -19,7 +19,7 @@ extra-source-files:
 # To avoid duplicated efforts in documentation and dealing with the
 # complications of embedding Haddock markup inside cabal files, it is
 # common to point users to the README.md file.
-description:         Please see the README on Github at <https://github.com/kframework/kore/blob/master/src/main/haskell/kore/README.md>
+description:         Please see the README on Github at <https://github.com/kframework/kore/blob/master/kore/README.md>
```

Comment:
It's really not necessary to even include the hyperlink, since `README.md` is part of the package.

Reply:
Using a local link to `README` instead of a global hyperlink.